schemas/com.roku/device_info/jsonschema/1-0-0

```diff
@@ -0,0 +1,159 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Context Schema with information about Roku device",
+  "self": {
+    "format": "jsonschema",
+    "name": "device_info",
+    "vendor": "com.roku",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "model": {
+      "type": "string",
+      "description": "Model name of the Roku device.",
+      "maxLength": 255
+    },
+    "modelDisplayName": {
+      "type": "string",
+      "description": "Model display name.",
+      "maxLength": 65535
+    },
+    "modelType": {
+      "type": "string",
+      "description": "Type of device.",
+      "maxLength": 255
+    },
+    "osVersion": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "Major, minor, revision, and build number of the Roku OS running on the device separated by dots.",
```

Comment:
Could add some regex here if we wanted to, although we'd have to be certain Roku won't suddenly change their os versioning schema so maybe not worth the risk.

Reply:
I think that's a good idea. There shouldn't be much risk in Roku changing their versioning schema because we control how this version string looks – the system call gives me a dictionary with keys for major, minor, revision and build number and I construct this version string in tracker code. I added a pattern regex to check for it.

Reply:
The more I look at this, the more nervous it makes me. What if someone does this with a beta release or something, could that introduce non-numeric values? I'm now feeling like we should remove this regex, not sure it adds to much value vs the risk it introduces to cause all events to fail.

Reply:
That's probably right, better to be cautious about this – I removed the regex pattern from the osVersion property.